docs/en/guides/developer/setup.md

```
@@ -0,0 +1,392 @@
+```bash
+# Run with uvicorn, hot reload enabled
+uvicorn app.main:app --host 0.0.0.0 --port 8000 --reload
+```
+
+Access API documentation:
+- Swagger UI: http://localhost:8000/api/docs
+- ReDoc: http://localhost:8000/api/redoc
+
+#### Backend Directory Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)